frontend/js/radio.js

```diff
@@ -156,6 +188,62 @@ function change_channel(e) {
     playlistPoll = setInterval(check_playlist, 15000);
 }
 
+function populate_schedule() {
+    ajax_with_json("/schedule.json", function(response) {
+        let schedule_div = document.getElementById("schedule_div");
+        schedule_div.innerText = "";
+        schedule_div.innerHTML = "";
+
+        if (response.week_of !== undefined){
+            let sched_header = document.createElement("h1");
+            sched_header.innerHTML = "Scheduled Events for the Week of " + response.week_of;
+            schedule_div.appendChild(sched_header);
+            schedule_div.appendChild(document.createElement("br"));
+        }
+
+
+
+        let tbl   = document.createElement("table");
+        let tbody = document.createElement("tbody");
+
+        var days_of_week = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'];
+        
+        var scheduled_events = [];
+        for (var i = 0; i < 7; i++) {
+            scheduled_events.push('None');
+        }
+
+        if (response.dailies !== undefined){
+            for (var j = 0; j < response.dailies.length; j++) {
+                if (response.dailies[j].length > 0){
+                    scheduled_events[j] = response.dailies[j];
+                }
+            }
+        }
+
+        for (var i = 0; i < 7; i++) {
+            let row = tbody.insertRow(tbody.rows.length);
+            let day_cell = row.insertCell(0);
+            day_cell.innerText = days_of_week[i];
+            day_cell.style.width = "20%";
+
+            let sched_cell = row.insertCell(1);
+            sched_cell.innerText = scheduled_events[i];
+            sched_cell.style.width = "75%";
+
+            tbody.appendChild(row);
+        }
+
+        tbl.appendChild(tbody);
+        schedule_div.appendChild(tbl);
+
+        let explanation = document.createElement("h2");
+        explanation.innerHTML = "<br>All times are in UTC";
```

Comment:
`<h2><br>...</h2>` feels like a fairly hacky way to achieve whatever effect it is you want.  Is this really the best way?  Or would some CSS be better?